css/cssom-view/support/scroll-behavior.js

```diff
@@ -41,6 +41,10 @@ function waitForScrollEnd(elements) {
 }
 
 function resetScroll(scrollingElement) {
+  // Reattach the node, make sure it won't be affected by last scrolling.
+  var parentElement = scrollingElement.parentNode;
+  parentElement.removeChild(scrollingElement);
+  parentElement.appendChild(scrollingElement);
```

Comment:
Is it really necessary? I believe the spec says the instant scroll below will cancel any scroll in progress.